.github/workflows/hermetic_library_generation.yaml

```diff
@@ -15,10 +15,7 @@
 # downstream client libraries before they are released.
 name: Hermetic library generation upon generation config change through pull requests
 on:
-  workflow_dispatch:
   pull_request:
-    paths:
```

Comment:
Can you remind me what is the trigger now? I guess we don't need it because the logic [here](https://github.com/googleapis/google-cloud-java/blob/main/generation/hermetic_library_generation.sh#L84-L87) should cover all the cases?
I guess it is the same reason we don't allow manual invocation anymore, because manual invocation would not do anything due to no generation_config.yaml change?

Reply:
> Can you remind me what is the trigger now?

Whenever there's a pull request open/edited in the repo, the workflow will be triggered.

We don't need the `paths` since the script can handle it (the logic you mentioned). All we need is a pull request and manual invocation doesn't have one.

Reply:
Thanks, do you mind adding these info to the PR description?

Reply:
Done.